engine/engine.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto3";
+
+import "types/types.proto";
+import "google/protobuf/empty.proto";
+
+package engine;
+
+option go_package = "./engine;engine";
+
+// ------------------------------------------------------------------------
+// Engine API RPC requests natively implemented in the Erigon node backend
+// See https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md
+service Engine {
+  // Validate and possibly execute the payload.
+  rpc EngineNewPayload(types.ExecutionPayload) returns(EnginePayloadStatus);
+
+  // Update fork choice
+  rpc EngineForkChoiceUpdated(EngineForkChoiceUpdatedRequest) returns(EngineForkChoiceUpdatedResponse);
+
+  // Fetch Execution Payload using its ID.
+  rpc EngineGetPayload(EngineGetPayloadRequest) returns(EngineGetPayloadResponse);
+
+  rpc EngineGetPayloadBodiesByHashV1(EngineGetPayloadBodiesByHashV1Request) returns(EngineGetPayloadBodiesV1Response);
+
+  rpc EngineGetPayloadBodiesByRangeV1(EngineGetPayloadBodiesByRangeV1Request) returns(EngineGetPayloadBodiesV1Response);
+
+  // Fetch the blobs bundle using its ID.
+  rpc EngineGetBlobsBundleV1(EngineGetBlobsBundleRequest) returns(types.BlobsBundleV1);
+  // Should be moved to execution.proto
+  rpc PendingBlock(google.protobuf.Empty) returns (PendingBlockReply);
```

Comment:
Why is `PendingBlock` duplicated in engine & execution?

Reply:
So ideally PendingBlock should be part of Execution.proto at a conceptual level. However, as of now PendingBlock rely on engine logic. Therefore, I repeated it, it is also a nice way to remind myself to fix it later on the road.

Reply:
Remember: this is just separation of logic, I think the whole Engine GRPC interface is temporary as well as I want to move to json